free-tunes/Controllers/LoginViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  LoginViewController.swift
+//  free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/02/28.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+
+    @IBOutlet weak var headerText: UILabel!
+    @IBOutlet weak var usernameInput: UITextField!
+    @IBOutlet weak var passwordInput: UITextField!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupInputFields()
+        headerText.buildCustomLabel(labelFirstPart: "free", labelSecondPart: "Tunes")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        self.navigationController?.setNavigationBarHidden(false, animated: animated)
+        super.viewWillAppear(animated)
+    }
+
+    func setupInputFields() {
+        usernameInput.applyDefaultStyle(withName: "Username")
+        usernameInput.delegate = self
+        usernameInput.delegate = self
+        passwordInput.applyDefaultStyle(withName: "Password")
+        passwordInput.delegate = self
+        passwordInput.delegate = self
+    }
+
+    func loginRequest() -> Bool {
+        if usernameInput.text != "Admin" || passwordInput.text != "TestPass123" {
```

Comment:
This function is doing a little bit too much, maybe have a separate function where you do this check 😄

Reply:
I agree @bgouws